db/controllers/controllers.js

```diff
@@ -8,6 +8,8 @@ const {
   fetchUsers,
 } = require("../models/models.js");
 
+const endpoints = require("../../endpoints.json");
+
```

Comment:
Requiring in json files can be problematic as the data can become stale - this is why we recommend using the `fs` module and reading a file - HOWEVER - it's absolutely fine in this case as you never expect this file to change after it's creation - if you do update it in future, you'll upload the changes to your application anyway 👍